geo-data-server/src/test/java/com/powsybl/geodata/server/GeoDataApplicationTest.java

```diff
@@ -0,0 +1,35 @@
+package com.powsybl.geodata.server;
+
+import org.cassandraunit.spring.CassandraDataSet;
+import org.cassandraunit.spring.CassandraUnitDependencyInjectionTestExecutionListener;
+import org.cassandraunit.spring.CassandraUnitTestExecutionListener;
+import org.cassandraunit.spring.EmbeddedCassandra;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static org.junit.Assert.assertNotNull;
+import static org.springframework.test.context.TestExecutionListeners.MergeMode.MERGE_WITH_DEFAULTS;
+
+@RunWith(SpringRunner.class)
+@ContextConfiguration(classes = {GeoDataApplication.class})
+@TestExecutionListeners(listeners = {CassandraUnitDependencyInjectionTestExecutionListener.class,
+        CassandraUnitTestExecutionListener.class},
+        mergeMode = MERGE_WITH_DEFAULTS)
+@CassandraDataSet(value = "geo_data.cql", keyspace = CassandraConstants.KEYSPACE_GEO_DATA)
+@EmbeddedCassandra
+public class GeoDataApplicationTest {
```

Comment:
What is the purpose of this unit test? It has no added value (just one non null assert).